src/mscorlib/corefx/System/Globalization/CultureData.Windows.cs

```diff
@@ -607,5 +607,26 @@ private static CultureInfo[] EnumCultures(CultureTypes types)
         {
             throw new NotImplementedException();
         }
+
+        private static string GetConsoleFallbackName(string cultureName)
+        {
+            throw new NotImplementedException();
+        }
+
+        private bool IsSupplementalCustomCulture
+        {
+            get { throw new NotImplementedException(); }
+        }
+        
+        private bool IsWin32Installed
+        {
+            get { throw new NotImplementedException(); }
+        }
+        
+        private bool IsReplacementCulture
+        {
+            get { throw new NotImplementedException(); }
```

Comment:
The current Windows implementation under corefx is not used but we have it to make it in sync with corert. at some point when we need to support all netstandard 2.0 APIs in corert we have to go and port everything to corert which will show all such places we need to implement . I can log issue for that but this is going to be looked at anyway when we enable the functionality on corert